src/App.css

```diff
@@ -0,0 +1,13 @@
+.container {
+  max-width: 500px;
+  padding: 1%;
+  margin-top: -3%;
+  margin-left: auto;
+  margin-right: auto;
+  align-items: center;
+  justify-content: center;
```

Comment:
why align-items and justify-content ? Are these properties being applied. Explore a bit about flex box.